assets/src/settings-page/paired-url-structure.js

```diff
@@ -138,8 +138,8 @@ export function PairedUrlStructure( { focusedSection } ) {
 			<p dangerouslySetInnerHTML={
 				{ __html:
 					sprintf(
-						/* translators: 1: AMP Analytics docs URL */
-						__( 'When using the Transitional or Reader template modes, your site is in a “Paired AMP” configuration where your canonical URLs are non-AMP and then you have separate AMP versions of your pages with AMP-specific URLs. The structure of the paired AMP URL is not important, whether using a query parameter or path suffix. The use of a query parameter is the most compatible across various sites and it has the benefit of not resulting in a 404 if the AMP plugin is deactivated, so that is why it is the default. <em>Please note that changing the paired URL structure can cause AMP pages to disappear from search results until your site has been re-indexed.</em> So if the current structure of your paired AMP URLs works for you, there is no need to change. If you\'re migrating from another AMP plugin that used a different paired URL structure than the default, then you may want to change this setting. <a href="%1$s" target="_blank">Learn more</a>.', 'amp' ),
+						/* translators: 1: AMP Paired URL Structure documentation URL */
+						__( 'When using the Transitional or Reader template modes, your site is in a “Paired AMP” configuration. Canonical URLs are non-AMP and separate AMP versions of your pages have AMP-specific URLs. The structure of a paired AMP URL is not important, whether using a query parameter or path suffix. The use of a query parameter is the most compatible across various sites and doesn\'t result in a 404 if the AMP plugin is deactivated. <em>Note: Changing the paired URL structure can cause AMP pages to disappear from search results until your site is re-indexed.</em> If you\'re migrating from another AMP plugin with a different paired URL structure, then you may want to change this setting. Otherwise we recommend leaving it as is. <a href="%1$s" target="_blank">Learn more</a>', 'amp' ),
```

Comment:
```suggestion
						__( 'When using the Transitional or Reader template modes, your site is in a “Paired AMP” configuration. Your site's canonical URLs are non-AMP, and the separate AMP versions of your pages have AMP-specific URLs. The structure of a paired AMP URL is not important, whether using a query parameter or path suffix. The use of a query parameter is the most compatible across various sites and it has the benefit of not resulting in a 404 if the AMP plugin is deactivated. <em>Note: Changing the paired URL structure can cause AMP pages to temporarily disappear from search results until your site is re-indexed.</em> If you\'re migrating from another AMP plugin with a different paired URL structure, then you may want to change this setting. Otherwise we recommend leaving it as is. <a href="%1$s" target="_blank">Learn more</a>', 'amp' ),
```